packages/cross_file/README.md

```diff
@@ -27,16 +50,12 @@ You will find links to the API docs on the [pub page](https://pub.dev/packages/c
 
 ## Web Limitations
 
-`XFile` on the web platform is backed by [Blob](https://api.dart.dev/be/180361/dart-html/Blob-class.html)
+`XFile` on the web platform is backed by `Blob`
 objects and their URLs.
 
 It seems that Safari hangs when reading Blobs larger than 4GB (your app will stop
 without returning any data, or throwing an exception).
 
-This package will attempt to throw an `Exception` before a large file is accessed
```

Comment:
Just to be sure, was this comment removed, because we now refer to the browser's behavior? Or was this fixed in recent versions of Safari?

Reply:
We don't explicitly throw the "4 Gigabyte" exception; the browser may fail in unexpected ways (depending on what we end up doing with the bytes of the file)